config_templates/gretel/synthetics/tabular-differential-privacy.yml

```diff
@@ -0,0 +1,12 @@
+# Default configuration for Gretel Tabular DP to generate synthetic data with
+# differential privacy guarantees
+
+schema_version: "1.0"
+name: "tabular-differential-privacy"
+models:
+  - tabular_dp:
+      data_source: __tmp__
+      params:
+        epsilon: 1
+        delta: 0.0001
```

Comment:
These are fairly aggressive params as defaults. Epsilon of 1 and expecting 10000 rows of input data. I'd guess most uses will be first-timers and with not that much data.

Does the quality drop that much from using `delta=0.0001` versus `delta=0.001` to make it worth being a bit more lax here? Or do you think it just looks better to have aggressive default params?

Reply:
The reason for the `eps=1, delta = 0.0001` choice is good results with benchmark, where I varied epsilon and kept `delta=0.0001` fixed.

- I noticed the improvement from increasing epsilon to 2 is marginal for most datasets (in fact, some datasets had better performance with eps=0.1). I wanted to highlight the particularly impressive performance because `eps=1` is expected to be an "academic" value, rather than a practical one.
- Of course, decreasing delta will help improve accuracy of the DP methods contained in this model. However, we'll always want to encourage setting this to `1/n`. I would advocate for implementing `delta: auto` very soon and updating the recommended value in this config.

Reply:
Excellent, using `eps=1` as the default is great. And sounds like you can add the `delta: auto` support soon.